webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -109,9 +109,9 @@ class SubscriptionsPage extends Component {
   async pollTasks() {
     const { pollBulkSearch, organization } = this.props;
 
-    if (organization && organization.owner_details) {
+    if (organization) {
       pollBulkSearch({
-        action: `organization '${organization.owner_details.displayName}'`,
```

Comment:
I don't know what the intention of this code was. This filtering was not working as intended such that it was not finding in-progress tasks for the organization. Now it filters in a more succinct way - by org ID and action labels. Let me know if I'm missing the true intent of this method